control_msgs/action/AntipodalGripperCommand.action

```diff
@@ -0,0 +1,11 @@
+sensor_msgs/JointState command # The velocity and effort fields are optionally and not used if empty.
+---
+float64 position  # The current gripper gap size (in meters)
+float64 effort    # The current effort exerted (in Newtons)
+bool stalled      # True iff the gripper is exerting max effort and not moving
+bool reached_goal # True iff the gripper position has reached the commanded setpoint
+---
+float64 position  # The current gripper gap size (in meters)
+float64 effort    # The current effort exerted (in Newtons)
+bool stalled      # True iff the gripper is exerting max effort and not moving
+bool reached_goal # True iff the gripper position has reached the commanded setpoint
```

Comment:
If the gripper stalled and the goal was to close completely is the expected response here `false`?

Reply:
I think how it is supposed to work is that the gripper will be commanded to a position and either reaching the goal or stalling will terminate the action and respond. The client then can check if the gripper stalled or reached the position.